dcicutils/ff_utils.py

```diff
@@ -1040,19 +1041,25 @@ def get_indexing_status(key=None, ff_env=None):
     return _get_page(page='/indexing_status', key=key, ff_env=ff_env)
 
 
-def are_counts_even(env):
-    """ Returns 2-tuple of boolean on whether or not counts are even the value of the counts """
+def get_counts_summary(env):
+    """ Returns a named tuple given an FF name to check representing the counts state.
+            CountSummary
+                are_even: boolean on whether or not counts are even
+                summary_total: raw value of counts
+    """
+    CountSummary = namedtuple('CountSummary', ['are_even', 'summary_total'])
```

Comment:
I would probably put this at toplevel. It is effectively a type declaration. If you do it this way, you'll be going through the whole labor of defining and using a disposable type on every call (which actually does string manipulation and calls eval or execute to define the class, so can't easily be optimized by the JIT unless it has special knowledge of this class). But basically it means the type of any return value will not match the type of any other return value, so it's a bit of a hassle for the GC, too. Sorry if my earlier example confused you on this.